assets/templates/aws.ec2_logs/fields.yml

```diff
@@ -0,0 +1,15 @@
+- name: timestamp
+  # TIMESTAMP_ISO8601 %{YEAR}-%{MONTHNUM}-%{MONTHDAY}[T ]%{HOUR}:?%{MINUTE}(?::?%{SECOND})?%{ISO8601_TIMEZONE}?
+  type: date
+- name: syslog_timestamp
+  # SYSLOGTIMESTAMP %{MONTH} +%{MONTHDAY} %{TIME}
+  type: date
+- name: iporhost
+  # This can also be hostname, can we express it?
+  type: ip
```

Comment:
@aspacca Do you know how can we generate appropriate data for this "ip or host" field? `host` can be a `keyword` field, but how may we generate both data types?

Reply:
we don't have a way directly from fields/config yaml

it can be done directly in the template, at least with gotext. something like:
```
{{- $ip := generate "ip" }}
{{- $host := generate "host" }}
{{- $hostOrIp := generate "hostOrIp" }} # this a boolean field
{{ ternary $host $ip $hostOrIp }}
```